ui/src/kapacitor/containers/KapacitorRulePage.js

```diff
@@ -69,36 +62,31 @@ class KapacitorRulePage extends Component {
       addFlashMessage,
       queryConfigActions,
     } = this.props
-    const {enabledAlerts, kapacitor} = this.state
-    const rule = this.isEditing()
-      ? rules[params.ruleID]
-      : rules[DEFAULT_RULE_ID]
+    const {handlersFromConfig, kapacitor} = this.state
+    const rule =
+      params.ruleID === 'new' ? rules[DEFAULT_RULE_ID] : rules[params.ruleID]
```

Comment:
just in case, here's the final instance of `'new'` that i'm seeing

Reply:
'new's are not a problem in themselves.